src/main/java/org/opensearch/flowframework/transport/DeprovisionWorkflowTransportAction.java

```diff
@@ -246,7 +242,7 @@ private void updateWorkflowState(
                 Map.ofEntries(
                     Map.entry(STATE_FIELD, State.COMPLETED),
                     Map.entry(PROVISIONING_PROGRESS_FIELD, ProvisioningProgress.DONE),
-                    Map.entry(PROVISION_START_TIME_FIELD, Instant.now().toEpochMilli()),
+                    Map.entry(PROVISION_END_TIME_FIELD, Instant.now().toEpochMilli()),
```

Comment:
is there a new provision end time when we fail deprovisioning? Is the end time here not when the provisioning itself ends (last step is executed), unrelated to deprovisioning?